src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -233,20 +233,31 @@ public function assets() {
 
 		tribe_asset(
 			$plugin,
-			'tribe-tickets-rsvp',
-			'v2/rsvp.js',
-			[ 'jquery' ],
+			'tribe-tickets-rsvp-manager',
+			'v2/rsvp-manager.js',
+			[ 'jquery', 'tribe-common', 'tribe-tickets-rsvp-block' ],
 			null,
 			[
 				'localize' => [
 					'name' => 'TribeRsvp',
 					'data' => [
 						'ajaxurl' => admin_url( 'admin-ajax.php', ( is_ssl() ? 'https' : 'http' ) ),
+						'cancelText' => esc_js( 'Are you sure you want to cancel?', 'event-tickets' )
 					],
 				],
+				'groups' => 'tribe-tickets-rsvp',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 3 space(s) between "'groups'" and double arrow, but found 1.

<sub>hash: e7001810fedeba28917ce4a351f5c682</sub>